src/generate.rs

```diff
@@ -504,7 +540,7 @@ type = "remove_fields"
         );
 
         assert_eq!(
-            generate_example(false, "/add_fields,json_parser,remove_fields"),
+            generate_example(false, "/add_fields,json_parser,remove_fields", &None),
```

Comment:
It would be good to test it actually generating a file or two, making sure it creates what it is supposed to, etc.

Reply:
Yeah I wasn't sure on this because of a lack of personal context on CI etc.. Its an odd test since it would have to manipulate a local file system for verification. If everyone is good with that and it won't cause any trouble for CI I'd be happy to add a test! :smiley:

Reply:
Yup in general feel free to do this as long as you clean up in tempdirs.